requirements.txt

```diff
@@ -6,3 +6,5 @@ pandas>=2.0.3
 pyarrow>=15
 scikit-learn>=1.2.1
 scipy>=1.9.3
+cvxpy
```

Comment:
This may be a problem... I believe this has come up in the past, too. I wouldn't mind it as a soft dependency. Paging @fairlearn/fairlearn-maintainers 🙂